### ISSUE TITLE
Choosing subsets from palette

### ISSUE THREAD

**Issue by oddmentations** [karthik/wesanderson#32]
I am wondering if there is a way to choose a specific subset from a palette. For example, if I am using **GrandBudapest2** and would like to use the **first** and **second** color in `scale_fill_manual` and then use the **third** and **fourth** from the same palette in `scale_color_manual`. Is this possible? 

Thank you!

**Comment by karthik** [comment]
Yes totally doable. You can just subset colors into new palettes and use them as needed. If you need more help, please share a reproducible example.

**Comment by kbroman** [comment]
@informavorous Try using `wes_palettes`, which is a list of vectors of colors. Something like this:

```r
library(wesanderson)
gp2 <- wes_palettes$GrandBudapest2

ggplot( .... ) + .... + 
    scale_color_manual(values=gp2[3:4]) + 
    scale_fill_manual(values=gp2[1:2])
```

**Comment by informavorous** [comment]
Thank you!

**Comment by larissabr** [comment]
How could I select specific colours from a palette? Let's say I have a line chart with two lines but would like the last two colours rather than the first two in a palette. (The above didn't work). Thanks!
